src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/EqualityConstraintValidation.feature.disabled

```diff
@@ -0,0 +1,192 @@
+Feature: Equality Constraint Validation
```

Comment:
File shouldn't be disabled because there are some used scenarios.

Reply:
Renamed